### PR TITLE
Merge 19.09 into dev

### PR DESCRIPTION
**Pull request by nsoranzo**

### CODE REVIEW

**Comment by nsoranzo** [comment]
Both selenium and integration test failures seem legit, ping @dannon @mvdbeek

**Comment by dannon** [comment]
@nsoranzo thanks for the ping, I'll take a look

File: client/galaxy/scripts/components/login/Register.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <div class="container">
         <div class="row justify-content-md-center">
             <div class="col" :class="{ 'col-lg-6': !isAdmin }">
-                <b-alert :show="showRegistrationWarning" variant="danger">
+                <b-alert :show="showRegistrationWarning" variant="info">
```
Comment (martenson) [inline client/galaxy/scripts/components/login/Register.vue:5]: 👍

**Comment by mvdbeek** [comment]
Thanks @nsoranzo!

**Pull request merged**